docs/source/tutorials/ATL03/IceSat_ATL03_Access.ipynb

```diff
@@ -0,0 +1,781 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.    gt1l_df = pd.DataFrame({'Latitude': gt1l_lat, 'Longitude': gt1l_lon, 'Photon_Height': gt1l_height,'Along_track_distance':gt1l_dist_ph})
</pre><p>In the next iteration it would be good to make this a geopandas dataframe, not just pandas.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/150/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>